markers/rhv.py

```diff
@@ -0,0 +1,22 @@
+"""rhv(tier_number): Run only tests marked with RHV tier, e.g. rhv2.
+
+This marker is used for purposes of RHV - CFME integration.
+
+Tests can be marked like this:
+
+.. code-block:: python
+
+    @pytest.mark.rhv2
+    def test_something():
+        assert True
+
+Usage on CLI::
+
+    pytest -m 'rhv1'  # Run only tier 1
+    pytest -m 'rhv1 or rhv2 or rhv3'  # Run all the tiers
+    pytest -m 'not rhv3'  # Run all test methods except for RHV tier 3
```

Comment:
1. Did you tried to run it is the test collection works as expected?
2 . How do I run Everything that is not marked with rhv1-3? - 'not rhv1 and not rhv2 and not rhv3' ?
    We will need it to run all the tests that are not marked yet. Please try it and update.

Reply:
ad 1) I tried several scenarios that occurred to me and they worked, but I admit I did not try to think of all the possible scenarios. Some might have eluded me. The thing is that if we find out we are missing something, we can expand on it later once the need arises.
ad 2) You are right, I tried it and it worker: pytest -m 'not rhv1 and not rhv2 and not rhv3'

Reply:
If we want to use this we should enforce it. 

1. We need to define what limitations we have for tier1-3, and how to decide which one the test case is
2. We need to mark all test cases with it, thus Daniels scenario would not occur

Reply:
I was thinking, it would be nice (though it would introduce more code in this plugin), to have it like this:
```python
@pytest.mark.rhv(2)
def test_foo():
```

And you would have a special parameter for tests like `--rhv-tier 2` which would only run `.mark.rhv(2)`. Or `--rhv-tier 1 --rhv-tier 2`.

Just throwing some ideas out, feel free to ignore me :)

Reply:
@mfalesni From the design point of view, I completely agree. However, from the practical point of view, I would not go this way. I am still not very familiar with pytest internals and this implementation would require quite a deep dive into them. That means it would take me significant amount of time to code it. But we need to have this functionality rather sooner than later. That's why I decided to go with solution that is not perfect from design point of view, but is quick and clear and (I hope so) not hacky.

Reply:
Jan +1
We are waiting for this merge for 10 days already and missed one Downstream Stable already
Please merge it @mfalesni